src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -325,6 +325,175 @@ std::optional<Values> Visitor::visit(Parser::ValuesClauseContext* ctx) {
+ParsedQuery Visitor::visit(Parser::Update1Context* ctx) {
+  if (ctx->modify()) {
+    return visit(ctx->modify());
+  }
+
+  parsedQuery_._clause = parsedQuery::UpdateClause();
+
+  if (ctx->insertData()) {
+    parsedQuery_.updateClause().toInsert_ = visit(ctx->insertData());
+  } else if (ctx->deleteData()) {
+    parsedQuery_.updateClause().toDelete_ = visit(ctx->deleteData());
+  } else {
+    visitAlternative<void>(ctx->load(), ctx->clear(), ctx->drop(), ctx->add(),
+                           ctx->move(), ctx->copy(), ctx->create(),
+                           ctx->deleteWhere());
+    AD_FAIL();
+  }
+
+  return parsedQuery_;
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::LoadContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Load is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::ClearContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Clear is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::DropContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Drop is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::CreateContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Create is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::AddContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Add is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::MoveContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Move is not supported.");
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::CopyContext* ctx) {
+  reportNotSupported(ctx, "SPARQL 1.1 Update Copy is not supported.");
+}
+
+// ____________________________________________________________________________________
+vector<SparqlTripleSimple> Visitor::visit(Parser::InsertDataContext* ctx) {
+  return visit(ctx->quadData());
+}
+
+// ____________________________________________________________________________________
+vector<SparqlTripleSimple> Visitor::visit(Parser::DeleteDataContext* ctx) {
+  return visit(ctx->quadData());
+}
+
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::DeleteWhereContext* ctx) {
+  // DeleteData only deletes the specified triples. DeleteWhere matches the
+  // given pattern and then deletes the matched triples. This is a shortcut for
+  // a modify query.
+  reportNotSupported(ctx, "SPARQL 1.1 Update DeleteWhere is not supported.");
+}
```

Comment:
Isn't this (and several other of the syntactic sugar cases) trivial to also do in this PR?

Reply:
Seems that I wrongly mentally grouped `Load` and `Clear` with the other graph management operations.

Reply:
`CLEAR` is there now. It does not make sense to add `LOAD` before named graphs strictly speaking. The graphs can only be loaded into explicitly named graphs and not into the `DEFAULT` graph (`CLEAR` can work on the `DEFAULT` graph which is the only support combination now). As there is also the added complexity of fetching and then loading the graph I would postpone `LOAD` to a later time.